test/testcafe/spec/EnrollAuthenticatorPhoneView_spec.js

```diff
@@ -138,50 +145,52 @@ test
+    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(false);
     await t.wait(30500);
-    await t.expect(challengePhonePageObject.resendEmailView().hasClass('hide')).notOk();
-    const resendEmailView = challengePhonePageObject.resendEmailView();
-    await t.expect(resendEmailView.innerText).eql('Haven\'t received an SMS? Send again');
+    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(true);
+    const resendCodeText = await enrollPhonePageObject.resendCodeText(1);
+    await t.expect(resendCodeText).contains('Haven\'t received an SMS?');
   });
 
 test
   .requestHooks(voiceMock)('Callout appears after 30 seconds in voice mode', async t => {
-    const challengePhonePageObject = await setup(t);
-    await challengePhonePageObject.clickNextButton();
-    await t.expect(challengePhonePageObject.resendEmailView().hasClass('hide')).ok();
+    const enrollPhonePageObject = await setup(t);
+    await enrollPhonePageObject.clickNextButton();
+    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(false);
     await t.wait(30500);
-    await t.expect(challengePhonePageObject.resendEmailView().hasClass('hide')).notOk();
-    const resendEmailView = challengePhonePageObject.resendEmailView();
-    await t.expect(resendEmailView.innerText).eql('Haven\'t received a call? Call again');
+    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(true);
+    const resendCodeText = await enrollPhonePageObject.resendCodeText(1);
+    await t.expect(resendCodeText).contains('Haven\'t received a call?');
   });
 
-test
+// Test fails in v3. After re-render we still have to wait for 30 seconds
+// Enable after fixing - OKTA-561098  
+test.meta('v3', false)
   .requestHooks(smsMock)('Callout appears after 30 seconds at most even after re-render', async t => {
-    const challengePhonePageObject = await setup(t);
-    await challengePhonePageObject.clickNextButton();
-    await t.expect(challengePhonePageObject.resendEmailView().hasClass('hide')).ok();
+    const enrollPhonePageObject = await setup(t);
+    await enrollPhonePageObject.clickNextButton();
+    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(false);
     await t.wait(15000);
-    challengePhonePageObject.navigateToPage();
-    await challengePhonePageObject.clickNextButton();
+    enrollPhonePageObject.navigateToPage();
+    await enrollPhonePageObject.clickNextButton();
     await t.wait(15500);
-    await t.expect(challengePhonePageObject.resendEmailView().hasClass('hide')).notOk();
-    const resendEmailView = challengePhonePageObject.resendEmailView();
-    await t.expect(resendEmailView.innerText).eql('Haven\'t received an SMS? Send again');
+    await t.expect(await enrollPhonePageObject.resendCodeExists(2)).eql(true);
+    const resendCodeText = await enrollPhonePageObject.resendCodeText(1);
+    await t.expect(resendCodeText).contains('Haven\'t received an SMS?');
```

Comment:
Can another assertion be placed to also check if the text contains `Send again`? 

I know in v3, the Send again link is not exactly concatenated with the `Haven't receieved an SMS?` part, but just wondering if we can still look for the text to make it pass for both v2 and v3?